src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -145,9 +147,36 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $io->listing($items);
         }
 
+        $list = array();
+        foreach ($this->getLoaderPaths() as $namespace => $paths) {
+            $list = array_merge($list, array_map(function ($path) use ($namespace) {
+                return $namespace.($namespace ? ': ' : '').$path;
+            }, $paths));
+        }
+        $io->section('Loader Paths');
+        $io->listing($list);
```

Comment:
Using a table might be a good alternative output to me.

Reply:
Agree.

Reply:
Done. (Btw, was necessary the `TableSeparator` and `*` to differentiate each row)